.travis.yml

```diff
@@ -1,7 +1,35 @@
 sudo: false
+
 language: node_js
+
 node_js:
-  - "stable"
   - "4"
-  - "0.12"
-  - "0.10"
```

Comment:
Nice commit!

But a lot of users still using Datauri over node 0.10, 0.12. Also, could you please change version 6 to "stable".